frontend/sass/styles.scss

```diff
@@ -256,8 +259,18 @@ fieldset {
         background: $light;
     }
 
-    li > div.jf-biglist-counter + div {
-        margin-left: 1em;
+    li > div.jf-biglist-counter {
+        // Align counters to top of their container:
+        margin-bottom: auto;
+        & + div {
+            margin-left: $jf-biglist-counter-margin;
+            // Make guidelines box extend outside of parent div on mobile:
+            @media screen and (max-width: $tablet) {
+                .jf-sanitation-guidelines {
+                    margin-left: calc(0px - #{$jf-biglist-counter-size} - #{$jf-biglist-counter-margin});
+                }
```

Comment:
This was something funky... Tahnee's design wanted the big Sanitation Guide banner to actually be full width on mobile devices. It seemed very disruptive to me to refactor your whole BigList component, so what I did was save the counter width and margin as scss variables, and then on mobile, I added a negative margin to the sanitation guide box to extend outside of its div according to these measurements. Seems to work fine but maybe a bit unorthodox. Let me know what you think.

Reply:
Oh interesting!  Huh, I think that's okay to do (I've definitely seen negative margins used in this kind of way before) but it's especially good that you're using the SASS variables here so it's very clear _why_ you're giving those negative margins. Noice!